pkg/event/target/amqp.go

```diff
@@ -30,18 +30,20 @@ import (
 
 // AMQPArgs - AMQP target arguments.
 type AMQPArgs struct {
-	Enable       bool     `json:"enable"`
-	URL          xnet.URL `json:"url"`
-	Exchange     string   `json:"exchange"`
-	RoutingKey   string   `json:"routingKey"`
-	ExchangeType string   `json:"exchangeType"`
-	DeliveryMode uint8    `json:"deliveryMode"`
-	Mandatory    bool     `json:"mandatory"`
-	Immediate    bool     `json:"immediate"`
-	Durable      bool     `json:"durable"`
-	Internal     bool     `json:"internal"`
-	NoWait       bool     `json:"noWait"`
-	AutoDeleted  bool     `json:"autoDeleted"`
+	Enable            bool       `json:"enable"`
+	URL               xnet.URL   `json:"url"`
+	Exchange          string     `json:"exchange"`
+	RoutingKey        string     `json:"routingKey"`
+	ExchangeType      string     `json:"exchangeType"`
+	DeliveryMode      uint8      `json:"deliveryMode"`
+	Mandatory         bool       `json:"mandatory"`
+	Immediate         bool       `json:"immediate"`
+	Durable           bool       `json:"durable"`
+	Internal          bool       `json:"internal"`
+	NoWait            bool       `json:"noWait"`
+	AutoDeleted       bool       `json:"autoDeleted"`
+	Arguments         amqp.Table `json:"arguments"`
+	PublishingHeaders amqp.Table `json:"publishingHeaders"`
```

Comment:
As new configuration elements are added, not sure whether its still valid to bump up config version and migration.